apigen/src/main/java/org/zaproxy/zap/extension/api/ApiGenerator.java

```diff
@@ -100,7 +102,8 @@ private static void generate(ApiImplementor api) {
                         wrapper(NodeJSAPIGenerator.class, NODE_OUTPUT_DIR),
                         wrapper(PhpAPIGenerator.class, PHP_OUTPUT_DIR),
                         wrapper(PythonAPIGenerator.class, PYTHON_OUTPUT_DIR),
-                        wrapper(RustAPIGenerator.class, RUST_OUTPUT_DIR)
+                        wrapper(RustAPIGenerator.class, RUST_OUTPUT_DIR),
+                        wrapper(DotNetAPIGenerator.class, DOTNET_OUTPUT_DIR)
```

Comment:
Could you add before `JavaAPIGenerator` to keep them in alphabetic order?

Reply:
Of course.